analysis/src/github.rs

```diff
@@ -317,11 +323,12 @@ impl GitHubAnalyzer {
         repo_fullname: &String,
         past_days: u64,
     ) -> Result<RecentActivity> {
+        let since_query_string = chrono::Utc::now()
+            .checked_sub_signed(chrono::Duration::days(past_days as i64))
+            .ok_or_else(|| anyhow!("Cannot convert past duration into query string"))?;
         let since_query_string =
-            match chrono::Utc::now().checked_sub_signed(chrono::Duration::days(past_days as i64)) {
-                Some(duration) => duration.to_rfc3339_opts(chrono::SecondsFormat::Secs, true),
-                None => return Err(anyhow!("Cannot convert past duration into query string")),
-            };
+            since_query_string.to_rfc3339_opts(chrono::SecondsFormat::Secs, true);
```

Comment:
You can use `.` after `?`:

```
let since_query_string = chrono::Utc::now()
    .check_sub_signed(...)
    .ok_or_else(...)?
    .to_rfc3339_opts(...);
```

Reply:
done. Thanks for the suggestion. @metajack